# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
msgid ""
msgstr ""
"Project-Id-Version: update-manager HEAD\n"
"Report-Msgid-Bugs-To: sebastian.heinlein@web.de\n"
"POT-Creation-Date: 2010-04-02 07:16+0000\n"
"PO-Revision-Date: 2010-03-31 10:25+0000\n"
"Last-Translator: cumulus007 <cumulus0007@gmail.com>\n"
"Language-Team: Nederlands <vertaling@vrijschrift.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Launchpad-Export-Date: 2010-04-07 08:08+0000\n"
"X-Generator: Launchpad (build Unknown)\n"

#. TRANSLATORS: download size is 0
#: ../DistUpgrade/utils.py:336 ../UpdateManager/Core/utils.py:336
msgid "0 KB"
msgstr "0 KB"

#. TRANSLATORS: download size of very small updates
#: ../DistUpgrade/utils.py:339 ../UpdateManager/Core/utils.py:339
msgid "1 KB"
msgstr "1 KB"

#. TRANSLATORS: download size of small updates, e.g. "250 KB"
#: ../DistUpgrade/utils.py:342 ../UpdateManager/Core/utils.py:342
#, python-format
msgid "%.0f KB"
msgstr "%.0f KB"

#. TRANSLATORS: download size of updates, e.g. "2.3 MB"
#: ../DistUpgrade/utils.py:345 ../UpdateManager/Core/utils.py:345
#, python-format
msgid "%.1f MB"
msgstr "%.1f MB"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:208 ../DistUpgrade/distro.py:423
#, python-format
msgid "Server for %s"
msgstr "Server voor %s"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:226 ../DistUpgrade/distro.py:232
#: ../DistUpgrade/distro.py:248
msgid "Main server"
msgstr "Hoofdserver"

#: ../DistUpgrade/distro.py:252
msgid "Custom servers"
msgstr "Andere servers"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:131
msgid "Could not calculate sources.list entry"
msgstr "Kon sources.list-vermelding niet berekenen"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:234
msgid ""
"Unable to locate any package files, perhaps this is not a Ubuntu Disc or the "
"wrong architecture?"
msgstr ""
"Kan geen pakketbestanden vinden, mogelijk is dit geen Ubuntu-medium of bevat "
"het niet de juiste architectuur."

#: ../DistUpgrade/DistUpgradeAptCdrom.py:277
msgid "Failed to add the CD"
msgstr "Toevoegen van de cd is mislukt"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:278
#, python-format
msgid ""
"There was a error adding the CD, the upgrade will abort. Please report this "
"as a bug if this is a valid Ubuntu CD.\n"
"\n"
"The error message was:\n"
"'%s'"
msgstr ""
"De upgrade is gestopt omdat de cd niet toegevoegd kon worden. Dien een "
"foutrapport in als dit wel een goede Ubuntu-cd is.\n"
"\n"
"De foutmelding was:\n"
"'%s'"

#: ../DistUpgrade/DistUpgradeCache.py:130
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "Verwijder pakket in slechte staat."
msgstr[1] "Verwijder pakketten in slechte staat."

#: ../DistUpgrade/DistUpgradeCache.py:133
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"Het pakket '%s' verkeert in een inconsistente staat en moet opnieuw "
"geïnstalleerd worden. Er is echter geen archiefbestand gevonden voor dit "
"pakket. Wilt u verdergaan en het pakket verwijderen?"
msgstr[1] ""
"De pakketten '%s' verkeren in een inconsistente staat en moeten opnieuw "
"geïnstalleerd worden. Er zijn echter geen archiefbestanden gevonden voor "
"deze pakketten. Wilt u verdergaan en de pakketten verwijderen?"

#. FIXME: not ideal error message, but we just reuse a
#. existing one here to avoid a new string
#: ../DistUpgrade/DistUpgradeCache.py:231
msgid "The server may be overloaded"
msgstr "De server is mogelijk overbelast"

#: ../DistUpgrade/DistUpgradeCache.py:341
msgid "Broken packages"
msgstr "Niet-werkende pakketten"

#: ../DistUpgrade/DistUpgradeCache.py:342
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"Uw systeem bevat niet-werkende pakketten die niet hersteld kunnen worden met "
"deze software. Herstel deze eerst met Synaptic of apt-get voordat u "
"verdergaat."

#. FIXME: change the text to something more useful
#: ../DistUpgrade/DistUpgradeCache.py:621
#, python-format
msgid ""
"An unresolvable problem occurred while calculating the upgrade:\n"
"%s\n"
"\n"
" This can be caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
" * Running the current pre-release version of Ubuntu\n"
" * Unofficial software packages not provided by Ubuntu\n"
"\n"
msgstr ""
"Een onoplosbaar probleem heeft zich voorgedaan tijdens het berekenen van de "
"upgrade:\n"
"%s\n"
"\n"
" Dit kan veroorzaakt worden door:\n"
"\n"
" * Upgraden naar een vooruitgave van Ubuntu\n"
" * Het gebruik van de huidige vooruitgave van Ubuntu\n"
" * Onofficiële softwarepakketten, niet aangeleverd door Ubuntu\n"
"\n"

#: ../DistUpgrade/DistUpgradeCache.py:631
msgid "This is most likely a transient problem, please try again later."
msgstr ""
"Dit is waarschijnlijk een tijdelijk probleem.\r\n"
"Probeer het later nog eens."

#: ../DistUpgrade/DistUpgradeCache.py:634
msgid ""
"If none of this applies, then please report this bug against the 'update-"
"manager' package and include the files in /var/log/dist-upgrade/ in the bug "
"report."
msgstr ""
"Als het bovenstaande niet van toepassing is, gelieve dan deze fout te "
"rapporteren in verband met het pakket 'update-manager', en voeg de bestanden "
"uit /var/log/dist-upgrade/ als bijlage aan het rapport toe."

#: ../DistUpgrade/DistUpgradeCache.py:640
#: ../UpdateManager/UpdateManager.py:766
msgid "Could not calculate the upgrade"
msgstr "Kan de vereisten voor de upgrade niet berekenen"

#: ../DistUpgrade/DistUpgradeCache.py:690
msgid "Error authenticating some packages"
msgstr "Fout bij het bepalen van de echtheid van sommige pakketten"

#: ../DistUpgrade/DistUpgradeCache.py:691
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"Het was niet mogelijk om de echtheid van sommige pakketten vast te stellen. "
"Dit kan liggen aan een tijdelijk netwerkprobleem. U kunt het later opnieuw "
"proberen. Hieronder vindt u een lijst met pakketten waarvan de echtheid niet "
"vastgesteld is."

#: ../DistUpgrade/DistUpgradeCache.py:711
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal blacklist."
msgstr ""
"Het pakket '%s' is gemarkeerd voor verwijdering, maar het staat op de zwarte "
"lijst voor verwijderen."

#: ../DistUpgrade/DistUpgradeCache.py:715
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "Het essentiële pakket '%s' is gemarkeerd voor verwijdering."

#: ../DistUpgrade/DistUpgradeCache.py:724
#, python-format
msgid "Trying to install blacklisted version '%s'"
msgstr ""
"Pakket (versie '%s') dat op de zwarte lijst staat proberen te installeren"

#: ../DistUpgrade/DistUpgradeCache.py:824
#, python-format
msgid "Can not mark '%s' for upgrade"
msgstr "Kan '%s' niet markeren om te upgraden"

#: ../DistUpgrade/DistUpgradeCache.py:838
#, python-format
msgid "Can't install '%s'"
msgstr "Kan '%s' niet installeren"

#: ../DistUpgrade/DistUpgradeCache.py:839
msgid ""
"It was impossible to install a required package. Please report this as a "
"bug. "
msgstr ""
"Het was niet mogelijk om een vereist pakket te installeren. Gelieve dit als "
"fout te rapporteren. "

#. FIXME: provide a list
#: ../DistUpgrade/DistUpgradeCache.py:848
msgid "Can't guess meta-package"
msgstr "Kan het metapakket niet raden"

#: ../DistUpgrade/DistUpgradeCache.py:849
msgid ""
"Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
"desktop or edubuntu-desktop package and it was not possible to detect which "
"version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""
"Uw systeem bevat geen van de volgende pakketten: ubuntu-desktop, kubuntu-"
"desktop of edubuntu-desktop.\n"
"Hierdoor is het niet mogelijk om uw versie van Ubuntu te bepalen.\n"
" Installeer eerst een van de bovenstaande pakketten met Synaptic of apt-get "
"voordat u verdergaat."

#: ../DistUpgrade/DistUpgradeController.py:86
msgid "Reading cache"
msgstr "Tijdelijke opslag inlezen"

#: ../DistUpgrade/DistUpgradeController.py:189
msgid "Unable to get exclusive lock"
msgstr "Kon geen exclusieve blokkering verkrijgen"

#: ../DistUpgrade/DistUpgradeController.py:190
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"Waarschijnlijk is er al een ander programma voor pakketbeheer (zoals apt-get "
"of aptitude) actief. Sluit dit programma eerst af."

#: ../DistUpgrade/DistUpgradeController.py:236
msgid "Upgrading over remote connection not supported"
msgstr "Upgraden over een externe verbinding wordt niet ondersteund"

#: ../DistUpgrade/DistUpgradeController.py:237
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. Please try a text mode upgrade with 'do-release-"
"upgrade'.\n"
"\n"
"The upgrade will abort now. Please try without ssh."
msgstr ""
"U voert de upgrade uit over een ssh-verbinding op afstand met een frontend "
"dat dit niet ondersteunt. Probeer een upgrade in tekstmodus met 'do-release-"
"upgrade'.\n"
"\n"
"De upgrade wordt nu afgebroken. Probeer het opnieuw zonder ssh."

#: ../DistUpgrade/DistUpgradeController.py:251
msgid "Continue running under SSH?"
msgstr "Doorgaan met uitvoeren via ssh?"

#: ../DistUpgrade/DistUpgradeController.py:252
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, an additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"Deze sessie lijkt onder ssh te draaien. Het wordt momenteel niet aangeraden "
"om een upgrade via ssh uit te voeren, omdat een mislukte upgrade dan "
"moeilijk te herstellen is.\n"
"\n"
"Als u verdergaat, zal er een extra ssh-daemon gestart worden op poort '%s'.\n"
"Wilt u verdergaan?"

#: ../DistUpgrade/DistUpgradeController.py:267
msgid "Starting additional sshd"
msgstr "Bezig met starten van extra sshd"

#: ../DistUpgrade/DistUpgradeController.py:268
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"Om bij fouten een hersteloperatie te vergemakkelijken, zal een extra sshd "
"gestart worden op poort ‘%s’. Indien er iets misgaat met de ssh die in "
"gebruik was, kunt u nog steeds verbinden met de extra sshd.\n"

#: ../DistUpgrade/DistUpgradeController.py:339
#: ../DistUpgrade/DistUpgradeController.py:384
msgid "Can not upgrade"
msgstr "Kan niet upgraden"

#: ../DistUpgrade/DistUpgradeController.py:340
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr "Een upgrade van '%s naar '%s' is niet mogelijk met dit programma."

#: ../DistUpgrade/DistUpgradeController.py:349
msgid "Sandbox setup failed"
msgstr "Sandbox-installatie mislukt"

#: ../DistUpgrade/DistUpgradeController.py:350
msgid "It was not possible to create the sandbox environment."
msgstr "Kon geen sandbox-omgeving aanmaken."

#: ../DistUpgrade/DistUpgradeController.py:356
msgid "Sandbox mode"
msgstr "Sandbox-modus"

#: ../DistUpgrade/DistUpgradeController.py:357
#, python-format
msgid ""
"This upgrade is running in sandbox (test) mode. All changes are written to "
"'%s' and will be lost on the next reboot.\n"
"\n"
"*No* changes written to a systemdir from now until the next reboot are "
"permanent."
msgstr ""
"Deze upgrade wordt uitgevoerd in de sandbox-modus (testmodus). Alle "
"wijzigingen worden geschreven naar '%s' en zullen verloren gaan bij een "
"herstart.\n"
"Er zullen *geen* veranderingen doorgevoerd worden in de systeemmappen, "
"totdat u de computer heeft herstart."

#: ../DistUpgrade/DistUpgradeController.py:385
msgid ""
"Your python install is corrupted. Please fix the '/usr/bin/python' symlink."
msgstr ""
"Uw python-installatie is beschadigd. Herstel de symbolische link "
"‘/usr/bin/python’."

#: ../DistUpgrade/DistUpgradeController.py:411
msgid "Package 'debsig-verify' is installed"
msgstr "Pakket 'debsig-verify' is geïnstalleerd."

#: ../DistUpgrade/DistUpgradeController.py:412
msgid ""
"The upgrade can not continue with that package installed.\n"
"Please remove it with synaptic or 'apt-get remove debsig-verify' first and "
"run the upgrade again."
msgstr ""
"De upgrade kan niet worden voortgezet als dat pakket geïnstalleerd is.\n"
"Verwijder het eerst met synaptic of 'apt-get remove debsig-verify' en voer "
"de upgrade opnieuw uit."

#: ../DistUpgrade/DistUpgradeController.py:422
msgid "Include latest updates from the Internet?"
msgstr "Laatste updates downloaden van het internet?"

#: ../DistUpgrade/DistUpgradeController.py:423
msgid ""
"The upgrade system can use the internet to automatically download the latest "
"updates and install them during the upgrade.  If you have a network "
"connection this is highly recommended.\n"
"\n"
"The upgrade will take longer, but when it is complete, your system will be "
"fully up to date.  You can choose not to do this, but you should install the "
"latest updates soon after upgrading.\n"
"If you answer 'no' here, the network is not used at all."
msgstr ""
"Bij de upgrade kunnen automatisch de laatste updates van het internet "
"gedownload en geïnstalleerd worden. Als u een netwerkverbinding heeft is dit "
"ten zeerste aan te bevelen.\n"
"\n"
"Het upgradeproces zal hierdoor langer duren, maar na afloop is uw systeem "
"geheel bijgewerkt. Als u hier niet voor kiest zult u snel na de upgrade de "
"updates moeten installeren.\n"
"Als u voor 'nee' kiest, wordt het netwerk helemaal niet gebruikt."

#: ../DistUpgrade/DistUpgradeController.py:647
#, python-format
msgid "disabled on upgrade to %s"
msgstr "uitgeschakeld bij upgrade naar %s"

#: ../DistUpgrade/DistUpgradeController.py:674
msgid "No valid mirror found"
msgstr "Geen geldige spiegelserver gevonden"

#: ../DistUpgrade/DistUpgradeController.py:675
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found. This can happen if you run a internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'No' the upgrade will cancel."
msgstr ""
"Tijdens het doorzoeken van uw beschikbare softwarebronnen is er geen "
"spiegelserver voor de upgrade gevonden. Dit kan gebeuren als u een interne "
"spiegelserver heeft of als de spiegelserverinformatie verouderd is.\n"
"\n"
"Wilt u 'sources.list' alsnog herschrijven? Als u 'Ja' kiest, worden "
"spiegelservers '%s' tot '%s' bijgewerkt. Bij 'Nee' wordt de upgrade "
"geannuleerd."

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:695
msgid "Generate default sources?"
msgstr "De standaard bronnenlijst aanmaken?"

#: ../DistUpgrade/DistUpgradeController.py:696
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""
"Tijdens het doorzoeken van uw beschikbare softwarebronnen is er geen "
"geschikte server voor '%s' gevonden.\n"
"\n"
"Moeten de standaardservers voor '%s' toegevoegd worden? Als u 'Nee' kiest, "
"wordt de upgrade geannuleerd."

#: ../DistUpgrade/DistUpgradeController.py:731
msgid "Repository information invalid"
msgstr "De informatie over de pakketbronnen is ongeldig"

#: ../DistUpgrade/DistUpgradeController.py:732
msgid ""
"Upgrading the repository information resulted in a invalid file. Please "
"report this as a bug."
msgstr ""
"Het upgraden van de informatie over de pakketbronnen heeft het bestand "
"ongeldig gemaakt. Rapporteer dit als een fout."

#: ../DistUpgrade/DistUpgradeController.py:738
msgid "Third party sources disabled"
msgstr "Pakketbronnen van derden uitgeschakeld"

#: ../DistUpgrade/DistUpgradeController.py:739
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"Enkele pakketbronnen van derden in uw sources.list zijn uitgeschakeld. U "
"kunt ze na de upgrade weer inschakelen met het programma ‘software-"
"properties’ of met ‘synaptic’."

#: ../DistUpgrade/DistUpgradeController.py:779
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "Pakket in inconsistente staat"
msgstr[1] "Pakketten in inconsistente staat"

#: ../DistUpgrade/DistUpgradeController.py:782
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"Het pakket '%s' verkeert in een inconsistente staat en moet opnieuw "
"geïnstalleerd worden. Er zijn echter geen archiefbestanden gevonden voor "
"deze pakketten. Gelieve het pakket handmatig te herinstalleren of te "
"verwijderen uit het systeem."
msgstr[1] ""
"De pakketten '%s' verkeren in een inconsistente staat en moeten opnieuw "
"geïnstalleerd worden. Er zijn echter geen archiefbestanden gevonden voor "
"deze pakketten. Gelieve de pakketten handmatig te herinstalleren of te "
"verwijderen uit het systeem."

#: ../DistUpgrade/DistUpgradeController.py:830
msgid "Error during update"
msgstr "Fout tijdens het bijwerken"

#: ../DistUpgrade/DistUpgradeController.py:831
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"Tijdens de update heeft zich een probleem voorgedaan. De oorzaak is meestal "
"een netwerkprobleem. Controleer uw netwerkverbinding en probeer het opnieuw."

#. print "on_button_install_clicked"
#: ../DistUpgrade/DistUpgradeController.py:840
#: ../UpdateManager/UpdateManager.py:573
msgid "Not enough free disk space"
msgstr "Niet genoeg vrije schijfruimte"

#: ../DistUpgrade/DistUpgradeController.py:841
#, python-format
msgid ""
"The upgrade is now aborted. The upgrade needs a total of %s free space on "
"disk '%s'. Please free at least an additional %s of disk space on '%s'. "
"Empty your trash and remove temporary packages of former installations using "
"'sudo apt-get clean'."
msgstr ""
"De upgrade is onderbroken. Er is een totaal van %s vrije ruimte nodig op "
"schijf '%s'. Maak ten minste %s ruimte vrij op schijf '%s'. Leeg uw "
"prullenbak en verwijder pakketten van eerdere installaties met 'sudo apt-get "
"clean'."

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:869
#: ../DistUpgrade/DistUpgradeController.py:1607
msgid "Calculating the changes"
msgstr "Wijzigingen worden berekend"

#: ../DistUpgrade/DistUpgradeController.py:885
msgid ""
"Canonical Ltd. no longer provides support for the following software "
"packages. You can still get support from the community."
msgstr ""
"Canonical Ltd. geeft geen ondersteuning meer voor de volgende "
"softwarepakketten. U kunt wel nog ondersteuning van de gemeenschap "
"verkrijgen."

#: ../DistUpgrade/DistUpgradeController.py:892
msgid ""
"Canonical Ltd. no longer provides support for the following software "
"packages. You can still get support from the community.\n"
"\n"
"These packages will be suggested for removal at the end of the upgrade "
"because you have not enabled the community maintained software channel "
"(universe)."
msgstr ""
"Canonical Ltd. geeft geen ondersteuning meer voor de volgende "
"softwarepakketten. U kunt wel nog ondersteuning van de gemeenschap "
"verkrijgen.\n"
"\n"
"Aan het einde van de upgrade zal gevraagd worden deze pakketten te "
"verwijderen, omdat u de door de gemeenschap onderhouden softwarebron "
"(universe) niet heeft geactiveerd."

#: ../DistUpgrade/DistUpgradeController.py:900
msgid "Support for some applications ended"
msgstr "Ondersteuning voor bepaalde toepassingen is beëindigd."

#. ask the user if he wants to do the changes
#: ../DistUpgrade/DistUpgradeController.py:920
msgid "Do you want to start the upgrade?"
msgstr "Wilt u beginnen met de upgrade?"

#: ../DistUpgrade/DistUpgradeController.py:985
msgid "Upgrade canceled"
msgstr "Upgrade geannuleerd"

#: ../DistUpgrade/DistUpgradeController.py:986
msgid ""
"The upgrade cancels now and the original system state is restored. You can "
"resume the upgrade at a later time."
msgstr ""
"De upgrade wordt nu geannuleerd en de originele systeemstatus is hersteld. U "
"kunt de upgrade op een later tijdstip voortzetten."

#: ../DistUpgrade/DistUpgradeController.py:991
#: ../DistUpgrade/DistUpgradeController.py:1094
msgid "Could not download the upgrades"
msgstr "Kon de upgrades niet downloaden"

#: ../DistUpgrade/DistUpgradeController.py:992
msgid ""
"The upgrade is now aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far are kept."
msgstr ""
"De upgrade is onderbroken. Controleer uw internetverbinding of "
"installatiemedium, en probeer opnieuw. Reeds gedownloade bestanden blijven "
"bewaard."

#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:1049
#: ../DistUpgrade/DistUpgradeController.py:1082
#: ../DistUpgrade/DistUpgradeController.py:1187
msgid "Error during commit"
msgstr "Fout bij het toepassen"

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1051
#: ../DistUpgrade/DistUpgradeController.py:1084
#: ../DistUpgrade/DistUpgradeController.py:1226
msgid "Restoring original system state"
msgstr "De oorspronkelijke toestand wordt hersteld"

#: ../DistUpgrade/DistUpgradeController.py:1052
#: ../DistUpgrade/DistUpgradeController.py:1066
#: ../DistUpgrade/DistUpgradeController.py:1085
msgid "Could not install the upgrades"
msgstr "Kon de upgrades niet installeren"

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:1057
msgid ""
"The upgrade is now aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"De upgrade is onderbroken. Het systeem is mogelijk in een onstabiele "
"situatie. Een herstelprocedure wordt nu uitgevoerd (dpkg --configure -a)."

#: ../DistUpgrade/DistUpgradeController.py:1062
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"files in /var/log/dist-upgrade/ in the bug report.\n"
"%s"
msgstr ""
"\n"
"\n"
"Gelieve deze fout te rapporteren in verband met het pakket 'update-manager', "
"en voeg de bestanden uit /var/log/dist-upgrade/ als bijlage aan de "
"rapportering toe.\n"
"%s"

#: ../DistUpgrade/DistUpgradeController.py:1095
msgid ""
"The upgrade is now aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr ""
"De upgrade is onderbroken. Controleer uw internetverbinding of "
"installatiemedium, en probeer opnieuw. "

#: ../DistUpgrade/DistUpgradeController.py:1175
msgid "Remove obsolete packages?"
msgstr "Verouderde pakketten verwijderen?"

#: ../DistUpgrade/DistUpgradeController.py:1176
#: ../DistUpgrade/DistUpgrade.ui.h:22
msgid "_Keep"
msgstr "_Behouden"

#: ../DistUpgrade/DistUpgradeController.py:1176
msgid "_Remove"
msgstr "_Verwijderen"

#: ../DistUpgrade/DistUpgradeController.py:1188
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr ""
"Tijdens het opruimen heeft zich een probleem voorgedaan. Zie onderstaand "
"bericht voor meer informatie. "

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1264
msgid "Required depends is not installed"
msgstr "De vereiste afhankelijkheden zijn niet geïnstalleerd"

#: ../DistUpgrade/DistUpgradeController.py:1265
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "Het vereiste pakket ‘%s’ is niet geïnstalleerd. "

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1510
#: ../DistUpgrade/DistUpgradeController.py:1577
msgid "Checking package manager"
msgstr "Pakketbeheerder controleren"

#: ../DistUpgrade/DistUpgradeController.py:1515
msgid "Preparing the upgrade failed"
msgstr "Voorbereiden van de upgrade is mislukt"

#: ../DistUpgrade/DistUpgradeController.py:1516
msgid ""
"Preparing the system for the upgrade failed. Please report this as a bug "
"against the 'update-manager' package and include the files in /var/log/dist-"
"upgrade/ in the bug report."
msgstr ""
"De voorbereidingen voor de upgrade zijn mislukt. Gelieve deze fout te "
"rapporteren in verband met het pakket 'update-manager', en voeg de bestanden "
"uit /var/log/dist-upgrade/ als bijlage aan de rapportering toe."

#: ../DistUpgrade/DistUpgradeController.py:1532
msgid "Getting upgrade prerequisites failed"
msgstr "Het voorbereiden van de upgrade is mislukt"

#: ../DistUpgrade/DistUpgradeController.py:1533
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Please report this as a bug against the 'update-manager' package and include "
"the files in /var/log/dist-upgrade/ in the bug report."
msgstr ""
"Het systeem was niet in staat de voorwaarden voor de upgrade te achterhalen. "
"De upgrade wordt nu afgebroken en het systeem wordt hersteld naar de "
"originele staat.\n"
"\n"
"Gelieve deze fout te rapporteren in verband met het pakket 'update-manager', "
"en voeg de bestanden uit /var/log/dist-upgrade/ als bijlage aan de "
"rapportering toe."

#: ../DistUpgrade/DistUpgradeController.py:1563
msgid "Updating repository information"
msgstr "Updaten van de informatie over de pakketbronnen"

#: ../DistUpgrade/DistUpgradeController.py:1595
msgid "Invalid package information"
msgstr "Ongeldige pakketinformatie"

#: ../DistUpgrade/DistUpgradeController.py:1596
#, python-format
msgid ""
"After your package information was updated the essential package '%s' can "
"not be found anymore.\n"
"This indicates a serious error, please report this bug against the 'update-"
"manager' package and include the files in /var/log/dist-upgrade/ in the bug "
"report."
msgstr ""
"Nadat uw pakketinformatie werd bijgewerkt, kon het essentiële pakket '%s' "
"niet meer gevonden worden.\n"
"Dit is een indicatie van een ernstige fout. Gelieve deze fout te rapporteren "
"in verband met het pakket 'update-manager', en voeg de bestanden uit "
"/var/log/dist-upgrade/ als bijlage aan de rapportering toe."

#: ../DistUpgrade/DistUpgradeController.py:1613
#: ../DistUpgrade/DistUpgradeController.py:1661
msgid "Fetching"
msgstr "Ophalen"

#: ../DistUpgrade/DistUpgradeController.py:1619
#: ../DistUpgrade/DistUpgradeController.py:1665
msgid "Upgrading"
msgstr "Bezig met upgraden"

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:1624
#: ../DistUpgrade/DistUpgradeController.py:1667
#: ../DistUpgrade/DistUpgradeController.py:1674
#: ../DistUpgrade/DistUpgradeController.py:1685
msgid "Upgrade complete"
msgstr "Upgrade is voltooid"

#: ../DistUpgrade/DistUpgradeController.py:1625
#: ../DistUpgrade/DistUpgradeController.py:1668
#: ../DistUpgrade/DistUpgradeController.py:1675
msgid ""
"The upgrade is completed but there were errors during the upgrade process."
msgstr ""
"De upgrade is voltooid, maar er zijn fouten opgetreden tijdens het "
"upgradeproces."

#: ../DistUpgrade/DistUpgradeController.py:1632
msgid "Searching for obsolete software"
msgstr "Zoeken naar verouderde software"

#: ../DistUpgrade/DistUpgradeController.py:1637
msgid "System upgrade is complete."
msgstr "Systeemupgrade is voltooid."

#: ../DistUpgrade/DistUpgradeController.py:1686
msgid "The partial upgrade was completed."
msgstr "De gedeeltelijke upgrade is voltooid."

#: ../DistUpgrade/DistUpgradeQuirks.py:149
msgid "No ARMv6 CPU"
msgstr "Geen ARMv6-CPU"

#: ../DistUpgrade/DistUpgradeQuirks.py:150
msgid ""
"Your system uses an ARM CPU that is older than the ARMv6 architecture. All "
"packages in karmic were built with optimizations requiring ARMv6 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"Uw systeem gebruikt een ARM-CPU ouder dan de ARMv6-architectuur. Alle "
"pakketten in karmic zijn gecompileerd voor ARMv6 als de minimale "
"architectuur. Het is met uw hardware niet mogelijk om uw systeem te upgraden "
"naar een nieuwe versie van Ubuntu."

#: ../DistUpgrade/DistUpgradeQuirks.py:164
msgid "No init available"
msgstr "Geen init beschikbaar"

#: ../DistUpgrade/DistUpgradeQuirks.py:165
msgid ""
"Your system appears to be a virtualised environment without an init daemon, "
"e.g. Linux-VServer. Ubuntu 9.10 cannot function within this type of "
"environment, requiring an update to your virtual machine configuration "
"first.\n"
"\n"
"Are you sure you want to continue?"
msgstr ""
"Het lijkt erop dat uw systeem gevirtualiseerd is zonder een init-daemon, "
"bijvoorbeeld Linux-VServer. Ubuntu 9.10 kan niet functioneren in dit type "
"virtualisatie. U zult eerst de instellingen van uw virtuele machine moeten "
"aanpassen.\n"
"\n"
"Weet u zeker dat u wilt doorgaan?"

#. TRANSLATORS: this string is not displayed in the 9.10 upgrade
#. (but it will be used for 8.04 -> 10.04 LTS upgrades)
#: ../DistUpgrade/DistUpgradeQuirks.py:186
#: ../DistUpgrade/DistUpgradeQuirks.py:268
#: ../DistUpgrade/DistUpgradeQuirks.py:295
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr ""
"Mogelijk worden tijdens een upgrade de bureaubladeffecten minder, en worden "
"de prestaties bij games en andere grafisch intensieve programma's negatief "
"beïnvloed."

#: ../DistUpgrade/DistUpgradeQuirks.py:190
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 10.04.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Deze computer gebruikt momenteel het grafisch stuurprogramma 'fglrx' van "
"AMD. Voor Ubuntu 10.04 is geen stuurprogramma beschikbaar dat met uw "
"hardware werkt.\n"
"Wilt u doorgaan?"

#: ../DistUpgrade/DistUpgradeQuirks.py:272
#: ../DistUpgrade/DistUpgradeQuirks.py:299
msgid ""
"This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
"version of this driver is available that works with your video card in "
"Ubuntu 10.04.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Deze computer gebruikt momenteel het grafisch stuurprogramma 'nvidia' van "
"NVIDIA. Voor Ubuntu 10.04 is geen stuurprogramma beschikbaar dat met uw "
"hardware werkt.\n"
"Wilt u doorgaan?"

#: ../DistUpgrade/DistUpgradeQuirks.py:473
msgid "evms in use"
msgstr "evms wordt gebruikt"

#: ../DistUpgrade/DistUpgradeQuirks.py:474
msgid ""
"Your system uses the 'evms' volume manager in /proc/mounts. The 'evms' "
"software is no longer supported, please switch it off and run the upgrade "
"again when this is done."
msgstr ""
"Uw systeem gebruikt de volumebeheerder 'evms' in /proc/mounts. De software "
"'evms' wordt niet langer ondersteund, schakel hem uit en voer de upgrade "
"nogmaals uit wanneer dit gebeurd is."

#: ../DistUpgrade/DistUpgradeMain.py:43
msgid "Sandbox upgrade using aufs"
msgstr "Sandbox-upgrade via aufs"

#: ../DistUpgrade/DistUpgradeMain.py:45
msgid "Use the given path to search for a cdrom with upgradable packages"
msgstr ""
"Het opgegeven pad om te zoeken naar een cd met pakketten die opgewaardeerd "
"kunnen worden"

#: ../DistUpgrade/DistUpgradeMain.py:51
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"Te gebruiken frontend. Momenteel beschikbaar: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:54
msgid "*DEPRECATED* this option will be ignore"
msgstr "*Verouderd* deze optie zal worden genegeerd"

#: ../DistUpgrade/DistUpgradeMain.py:57
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr ""
"Een gedeeltelijke upgrade uitvoeren (sources.list wordt niet herschreven)"

#: ../DistUpgrade/DistUpgradeMain.py:59
msgid "Set datadir"
msgstr "Datamap instellen"

#. print "mediaChange %s %s" % (medium, drive)
#: ../DistUpgrade/DistUpgradeViewGtk.py:109
#: ../DistUpgrade/DistUpgradeViewKDE.py:190
#: ../UpdateManager/DistUpgradeFetcherKDE.py:148
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "Plaats '%s' in station '%s'"

#: ../DistUpgrade/DistUpgradeViewGtk.py:129
#: ../DistUpgrade/DistUpgradeViewKDE.py:204
msgid "Fetching is complete"
msgstr "Het ophalen is voltooid"

#: ../DistUpgrade/DistUpgradeViewGtk.py:141
#: ../DistUpgrade/DistUpgradeViewKDE.py:217
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "Ophalen bestand %li van %li met %sB/s"

#: ../DistUpgrade/DistUpgradeViewGtk.py:142
#: ../DistUpgrade/DistUpgradeViewGtk.py:277
#: ../DistUpgrade/DistUpgradeViewKDE.py:218
#: ../DistUpgrade/DistUpgradeViewKDE.py:358
#, python-format
msgid "About %s remaining"
msgstr "Ongeveer %s resterend"

#: ../DistUpgrade/DistUpgradeViewGtk.py:144
#: ../DistUpgrade/DistUpgradeViewKDE.py:220
#, python-format
msgid "Fetching file %li of %li"
msgstr "Ophalen van bestand %li van %li"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk.py:176
#: ../DistUpgrade/DistUpgradeViewKDE.py:254
msgid "Applying changes"
msgstr "Wijzigingen worden doorgevoerd"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk.py:201
#: ../DistUpgrade/DistUpgradeViewKDE.py:267
msgid "dependency problems - leaving unconfigured"
msgstr "vereistenproblemen - blijft niet ingesteld"

#: ../DistUpgrade/DistUpgradeViewGtk.py:206
#: ../DistUpgrade/DistUpgradeViewKDE.py:269
#, python-format
msgid "Could not install '%s'"
msgstr "Kon '%s' niet installeren"

#: ../DistUpgrade/DistUpgradeViewGtk.py:207
#: ../DistUpgrade/DistUpgradeViewKDE.py:270
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may not be in a working "
"state. Please consider submitting a bug report about it."
msgstr ""
"De upgrade zal doorgaan, maar het pakket '%s' zal mogelijk niet werken. "
"Overweeg er een foutrapport over in te dienen."

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk.py:223
#: ../DistUpgrade/DistUpgradeViewKDE.py:291
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"Het aangepaste configuratiebestand vervangen?\n"
"'%s'"

#: ../DistUpgrade/DistUpgradeViewGtk.py:224
#: ../DistUpgrade/DistUpgradeViewKDE.py:292
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr ""
"Wanneer u besluit dit configuratiebestand te vervangen door een nieuwere "
"versie zullen al uw gemaakte wijzigingen verloren gaan."

#: ../DistUpgrade/DistUpgradeViewGtk.py:237
#: ../DistUpgrade/DistUpgradeViewKDE.py:310
msgid "The 'diff' command was not found"
msgstr "De opdracht 'diff' is niet gevonden"

#: ../DistUpgrade/DistUpgradeViewGtk.py:442
#: ../DistUpgrade/DistUpgradeViewText.py:88
msgid "A fatal error occurred"
msgstr "Er is een ernstige fout opgetreden"

#: ../DistUpgrade/DistUpgradeViewGtk.py:443
msgid ""
"Please report this as a bug (if you haven't already) and include the files "
"/var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade is now aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Gelieve deze fout te rapporteren (als u dat nog niet gedaan heeft) in "
"verband met het pakket 'update-manager', en voeg de bestanden /var/log/dist-"
"upgrade/main.log en /var/log/dist-upgrade/apt.log als bijlage aan de "
"rapportering toe. De upgrade is nu afgebroken.\n"
"Uw originele pakkettenlijst (sources.list) is opgeslagen in "
"/etc/apt/sources.list.distUpgrade."

#: ../DistUpgrade/DistUpgradeViewGtk.py:460
msgid "Ctrl-c pressed"
msgstr "Ctrl-c ingedrukt"

#: ../DistUpgrade/DistUpgradeViewGtk.py:461
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr ""
"Dit zal de taak onderbreken waardoor uw systeem mogelijk niet meer correct "
"werkt. Weet u zeker dat u dit wilt doen?"

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk.py:606
msgid "To prevent data loss close all open applications and documents."
msgstr ""
"Sluit alle openstaande toepassingen en documenten om dataverlies te "
"voorkomen."

#: ../DistUpgrade/DistUpgradeViewGtk.py:620
#, python-format
msgid "<b>Downgrade %s</b>"
msgstr "<b>%s downgraden</b>"

#: ../DistUpgrade/DistUpgradeViewGtk.py:622
#: ../DistUpgrade/DistUpgradeViewKDE.py:758
#, python-format
msgid "Remove %s"
msgstr "%s verwijderen"

#: ../DistUpgrade/DistUpgradeViewGtk.py:627
#: ../DistUpgrade/DistUpgradeViewKDE.py:760
#, python-format
msgid "Remove (was auto installed) %s"
msgstr "%s verwijderen (werd automatisch geïnstalleerd)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:630
#: ../DistUpgrade/DistUpgradeViewKDE.py:762
#, python-format
msgid "Install %s"
msgstr "%s installeren"

#: ../DistUpgrade/DistUpgradeViewGtk.py:632
#: ../DistUpgrade/DistUpgradeViewKDE.py:764
#, python-format
msgid "Upgrade %s"
msgstr "%s upgraden"

#. change = QMessageBox.question(None, _("Media Change"), msg, QMessageBox.Ok, QMessageBox.Cancel)
#: ../DistUpgrade/DistUpgradeViewKDE.py:191
#: ../UpdateManager/DistUpgradeFetcherKDE.py:150
msgid "Media Change"
msgstr "Medium wisselen"

#: ../DistUpgrade/DistUpgradeViewKDE.py:322
msgid "Show Difference >>>"
msgstr "Verschillen weergeven >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:325
msgid "<<< Hide Difference"
msgstr "<<< Verschillen verbergen"

#: ../DistUpgrade/DistUpgradeViewKDE.py:539
msgid "Error"
msgstr "Foutmelding"

#: ../DistUpgrade/DistUpgradeViewKDE.py:551
msgid "&Cancel"
msgstr "&Annuleren"

#: ../DistUpgrade/DistUpgradeViewKDE.py:553
#: ../DistUpgrade/DistUpgradeViewKDE.py:792
msgid "&Close"
msgstr "Sl&uiten"

#: ../DistUpgrade/DistUpgradeViewKDE.py:599
msgid "Show Terminal >>>"
msgstr "Terminalvenster weergeven >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:602
msgid "<<< Hide Terminal"
msgstr "<<< Terminalvenster verbergen"

#: ../DistUpgrade/DistUpgradeViewKDE.py:682
msgid "Information"
msgstr "Informatie"

#: ../DistUpgrade/DistUpgradeViewKDE.py:732
#: ../DistUpgrade/DistUpgradeViewKDE.py:775
#: ../DistUpgrade/DistUpgradeViewKDE.py:778 ../DistUpgrade/DistUpgrade.ui.h:11
msgid "Details"
msgstr "Details"

#: ../DistUpgrade/DistUpgradeViewKDE.py:788
#: ../DistUpgrade/DistUpgradeViewText.py:206
msgid "Restart required"
msgstr "De computer moet herstart worden"

#: ../DistUpgrade/DistUpgradeViewKDE.py:788
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr "<b><big>Herstart het systeem om de upgrade te voltooien</big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:791 ../DistUpgrade/DistUpgrade.ui.h:25
#: ../data/glade/UpdateManager.ui.h:31
msgid "_Restart Now"
msgstr "_Nu herstarten"

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:803
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>De upgrade annuleren?</big></b>\n"
"\n"
"Het systeem kan instabiel raken als u de upgrade annuleert. U wordt met "
"nadruk geadviseerd om met de upgrade door te gaan."

#: ../DistUpgrade/DistUpgradeViewKDE.py:807
msgid "Cancel Upgrade?"
msgstr "Upgrade annuleren?"

#: ../DistUpgrade/DistUpgradeView.py:60
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li dag"
msgstr[1] "%li dagen"

#: ../DistUpgrade/DistUpgradeView.py:62
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li uur"
msgstr[1] "%li uur"

#: ../DistUpgrade/DistUpgradeView.py:64
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li minuut"
msgstr[1] "%li minuten"

#: ../DistUpgrade/DistUpgradeView.py:65
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li seconde"
msgstr[1] "%li seconden"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:78
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s %(str_hours)s"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:93
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:128
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem."
msgstr ""
"Deze download zal ongeveer %s duren met een 1 Mbit DSL-verbinding en "
"ongeveer %s met een 56k-modem."

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:132
#, python-format
msgid "This download will take about %s with your connection. "
msgstr "Deze download duurt met uw verbinding ongeveer %s. "

#: ../DistUpgrade/DistUpgradeView.py:234 ../DistUpgrade/DistUpgrade.ui.h:16
msgid "Preparing to upgrade"
msgstr "Upgrade voorbereiden"

#: ../DistUpgrade/DistUpgradeView.py:235
msgid "Getting new software channels"
msgstr "Nieuwe softwarekanalen worden opgehaald"

#: ../DistUpgrade/DistUpgradeView.py:236 ../DistUpgrade/DistUpgrade.ui.h:14
msgid "Getting new packages"
msgstr "Ophalen van nieuwe pakketten"

#: ../DistUpgrade/DistUpgradeView.py:237 ../DistUpgrade/DistUpgrade.ui.h:15
msgid "Installing the upgrades"
msgstr "Installeren van de upgrades"

#: ../DistUpgrade/DistUpgradeView.py:238 ../DistUpgrade/DistUpgrade.ui.h:10
msgid "Cleaning up"
msgstr "Opruimen"

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:322
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "%d pakket zal verwijderd worden."
msgstr[1] "%d pakketten zullen verwijderd worden."

#: ../DistUpgrade/DistUpgradeView.py:327
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "%d nieuw pakket zal geïnstalleerd worden."
msgstr[1] "%d nieuwe pakketten zullen geïnstalleerd worden."

#: ../DistUpgrade/DistUpgradeView.py:333
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "%d pakket zal een upgrade krijgen."
msgstr[1] "%d pakketten zullen een upgrade krijgen."

#: ../DistUpgrade/DistUpgradeView.py:338
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"U moet in totaal %s downloaden. "

#: ../DistUpgrade/DistUpgradeView.py:342
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be cancelled."
msgstr ""
"Het upgraden kan enkele uren in beslag nemen en kan tussentijds niet "
"geannuleerd worden."

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:348 ../UpdateManager/UpdateManager.py:503
msgid "Your system is up-to-date"
msgstr "Uw systeem is up-to-date"

#: ../DistUpgrade/DistUpgradeView.py:349
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr ""
"Er zijn geen upgrades beschikbaar voor uw systeem. De upgrade wordt nu "
"geannuleerd."

#: ../DistUpgrade/DistUpgradeView.py:362
msgid "Reboot required"
msgstr "U moet de computer herstarten"

#: ../DistUpgrade/DistUpgradeView.py:363
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr ""
"Het upgraden is voltooid en herstarten is noodzakelijk. Wilt u dit nu doen?"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:135
#: ../DistUpgrade/DistUpgradeFetcherCore.py:136
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:135
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:136
msgid "Could not run the upgrade tool"
msgstr "Kon het upgradeprogramma niet uitvoeren"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:136
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:136
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug"
msgstr ""
"Dit is waarschijnlijk een fout in het upgradeprogramma. Rapporteer deze fout."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:205
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:205
msgid "Upgrade tool signature"
msgstr "Handtekening upgradeprogramma"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:208
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:208
msgid "Upgrade tool"
msgstr "Upgradeprogramma"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:237
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:237
msgid "Failed to fetch"
msgstr "Ophalen is mislukt"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:238
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:238
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr ""
"Het ophalen van de upgrade is mislukt. Er is mogelijk een netwerkprobleem. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:242
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:242
msgid "Failed to extract"
msgstr "Uitpakken is mislukt"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:243
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:243
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr ""
"Het uitpakken van de upgrade is mislukt. Er is mogelijk een probleem met het "
"netwerk of de server. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:248
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:248
msgid "Verification failed"
msgstr "Verificatie is mislukt"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:249
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:249
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr ""
"Het verifiëren van de upgrade is mislukt. Er is mogelijk een probleem met "
"het netwerk of de server. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:254
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:254
msgid "Authentication failed"
msgstr "Echtheidscontrole is mislukt"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:255
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:255
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr ""
"De echtheidscontrole van de upgrade is mislukt. Er is mogelijk een probleem "
"met het netwerk of de server. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:269
#: ../DistUpgrade/DistUpgradeFetcherCore.py:275
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:269
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:275
msgid "Can not run the upgrade"
msgstr "Kan de upgrade niet uitvoeren"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:270
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:270
msgid ""
"This usually is caused by a system were /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""
"Dit wordt meestal veroorzaakt door een system waar /tmp aangekoppeld is als "
"‘noexec’. Koppel de partitie opnieuw aan zonder ‘noexec’ en voer de upgrade "
"opnieuw uit."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:276
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:276
#, python-format
msgid "The error message is '%s'."
msgstr "De foutmelding is ‘%s’."

#: ../DistUpgrade/DistUpgradeViewText.py:89
msgid ""
"Please report this as a bug and include the files /var/log/dist-"
"upgrade/main.log and /var/log/dist-upgrade/apt.log in your report. The "
"upgrade is now aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Gelieve deze fout te rapporteren in verband met het pakket 'update-manager', "
"en voeg de bestanden /var/log/dist-upgrade/main.log en /var/log/dist-"
"upgrade/apt.log als bijlage aan de rapportering toe. De upgrade is nu "
"afgebroken.\n"
"Uw originele pakkettenlijst (sources.list) is opgeslagen in "
"/etc/apt/sources.list.distUpgrade."

#: ../DistUpgrade/DistUpgradeViewText.py:113
msgid "Aborting"
msgstr "Afbreken"

#: ../DistUpgrade/DistUpgradeViewText.py:118
msgid "Demoted:\n"
msgstr "Gedegradeerd:\n"

#: ../DistUpgrade/DistUpgradeViewText.py:150
#: ../DistUpgrade/DistUpgradeViewText.py:172
#: ../DistUpgrade/DistUpgradeViewText.py:179
msgid "Continue [yN] "
msgstr "_Doorgaan [jN] "

#: ../DistUpgrade/DistUpgradeViewText.py:150
#: ../DistUpgrade/DistUpgradeViewText.py:172
msgid "Details [d]"
msgstr "Details [d]"

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeViewText.py:154
#: ../DistUpgrade/DistUpgradeViewText.py:182
msgid "y"
msgstr "j"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:157
#: ../DistUpgrade/DistUpgradeViewText.py:189
msgid "n"
msgstr "n"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:160
msgid "d"
msgstr "d"

#: ../DistUpgrade/DistUpgradeViewText.py:164
#, python-format
msgid "Remove: %s\n"
msgstr "Verwijderen: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:167
#, python-format
msgid "Install: %s\n"
msgstr "Installeren: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:170
#, python-format
msgid "Upgrade: %s\n"
msgstr "Upgraden: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:186
msgid "Continue [Yn] "
msgstr "Doorgaan [Jn] "

#: ../DistUpgrade/DistUpgradeViewText.py:207
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"Om de upgrade te voltooien moet u de computer herstarten.\n"
"Wanneer u 'j' selecteert zal de computer herstart worden."

#: ../DistUpgrade/DistUpgrade.ui.h:1
msgid "    "
msgstr "    "

#: ../DistUpgrade/DistUpgrade.ui.h:2
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>De actieve upgrade annuleren?</big></b>\n"
"\n"
"Het systeem kan zich in een onbruikbare staat bevinden als u de upgrade "
"annuleert. U wordt sterk aangeraden om door te gaan met de upgrade."

#: ../DistUpgrade/DistUpgrade.ui.h:5
msgid ""
"<b><big>Restart the system to complete the upgrade</big></b>\n"
"\n"
"Please save your work before continuing."
msgstr ""
"<b><big>Herstart de computer om de upgrade te voltooien</big></b>\n"
"\n"
"Sla uw werk op alvorens u verdergaat."

#: ../DistUpgrade/DistUpgrade.ui.h:8
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>Upgrade starten?</big></b>"

#: ../DistUpgrade/DistUpgrade.ui.h:9
msgid "<b><big>Upgrading Ubuntu to version 10.04 LTS</big></b>"
msgstr "<b><big>Ubuntu upgraden naar versie 10.04 LTS</big></b>"

#: ../DistUpgrade/DistUpgrade.ui.h:12
msgid "Difference between the files"
msgstr "Verschillen tussen de bestanden"

#: ../DistUpgrade/DistUpgrade.ui.h:13
msgid "Distribution Upgrade"
msgstr "Distributie-upgrade"

#: ../DistUpgrade/DistUpgrade.ui.h:17
msgid "Restarting the computer"
msgstr "Computer herstarten"

#: ../DistUpgrade/DistUpgrade.ui.h:18
msgid "Setting new software channels"
msgstr "Instellen van de nieuwe softwarekanalen"

#: ../DistUpgrade/DistUpgrade.ui.h:19
msgid "Terminal"
msgstr "Terminalvenster"

#: ../DistUpgrade/DistUpgrade.ui.h:20
msgid "_Cancel Upgrade"
msgstr "Upgrade _annuleren"

#: ../DistUpgrade/DistUpgrade.ui.h:21
msgid "_Continue"
msgstr "_Doorgaan"

#: ../DistUpgrade/DistUpgrade.ui.h:23
msgid "_Replace"
msgstr "_Vervangen"

#: ../DistUpgrade/DistUpgrade.ui.h:24
msgid "_Report Bug"
msgstr "_Fout rapporteren"

#: ../DistUpgrade/DistUpgrade.ui.h:26
msgid "_Resume Upgrade"
msgstr "_Upgrade hervatten"

#: ../DistUpgrade/DistUpgrade.ui.h:27
msgid "_Start Upgrade"
msgstr "Upgrade _starten"

#: ../UpdateManager/backend/InstallBackendSynaptic.py:42
msgid "Please wait, this can take some time."
msgstr "Een ogenblik geduld, dit kan even duren."

#: ../UpdateManager/backend/InstallBackendSynaptic.py:44
msgid "Update is complete"
msgstr "De update is voltooid"

#: ../UpdateManager/DistUpgradeFetcher.py:85
#: ../UpdateManager/DistUpgradeFetcherKDE.py:103
msgid "Could not find the release notes"
msgstr "Kon de versie-informatie niet vinden"

#: ../UpdateManager/DistUpgradeFetcher.py:86
#: ../UpdateManager/DistUpgradeFetcherKDE.py:104
msgid "The server may be overloaded. "
msgstr "De server is waarschijnlijk overbelast. "

#: ../UpdateManager/DistUpgradeFetcher.py:96
#: ../UpdateManager/DistUpgradeFetcherKDE.py:108
msgid "Could not download the release notes"
msgstr "Kon de versie-informatie niet downloaden"

#: ../UpdateManager/DistUpgradeFetcher.py:97
#: ../UpdateManager/DistUpgradeFetcherKDE.py:109
msgid "Please check your internet connection."
msgstr "Controleer uw internetverbinding."

#: ../UpdateManager/DistUpgradeFetcherKDE.py:62
#: ../UpdateManager/DistUpgradeFetcherKDE.py:85
msgid "Upgrade"
msgstr "Upgraden"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:89
#: ../data/glade/UpdateManager.ui.h:12
#: ../data/glade/UpgradePromptDialog.ui.h:4
msgid "Release Notes"
msgstr "Versie-informatie"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:128
#: ../UpdateManager/DistUpgradeFetcherKDE.py:141
#: ../UpdateManager/DistUpgradeFetcherKDE.py:143
msgid "Downloading additional package files..."
msgstr "Aanvullende pakketbestanden downloaden…"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:141
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "Bestand %s van %s met %sB/s"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:143
#, python-format
msgid "File %s of %s"
msgstr "Bestand %s van %s"

#: ../UpdateManager/ChangelogViewer.py:70
msgid "Open Link in Browser"
msgstr "Koppeling in browser openen"

#: ../UpdateManager/ChangelogViewer.py:72
msgid "Copy Link to Clipboard"
msgstr "Koppeling naar klembord kopiëren"

#: ../UpdateManager/GtkProgress.py:116
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr "Bestand %(current)li van %(total)li met %(speed)s/s downloaden"

#: ../UpdateManager/GtkProgress.py:121
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "Bestand %(current)li van %(total)li downloaden"

#: ../UpdateManager/UpdateManager.py:304
#, python-format
msgid "Version %s: \n"
msgstr "Versie %s: \n"

#: ../UpdateManager/UpdateManager.py:361
msgid "Downloading list of changes..."
msgstr "Lijst met wijzigingen downloaden…"

#: ../UpdateManager/UpdateManager.py:401
msgid "_Uncheck All"
msgstr "Alles _deselecteren"

#: ../UpdateManager/UpdateManager.py:407
msgid "_Check All"
msgstr "Alles _controleren"

#: ../UpdateManager/UpdateManager.py:451
#, python-format
msgid ""
"Download size: %s\n"
"%s selected."
msgstr ""
"Downloadgrootte: %s\n"
"%s geselecteerd."

#: ../UpdateManager/UpdateManager.py:456
msgid "Unknown download size"
msgstr "Downloadgrootte onbekend"

#: ../UpdateManager/UpdateManager.py:464
msgid ""
"It is unknown when the package information was updated last. Please try "
"clicking on the 'Check' button to update the information."
msgstr ""
"Het is niet bekend wanneer de pakketinformatie voor het laatst werd "
"bijgewerkt. Klik op de knop 'Controleren' om de informatie bij te werken."

#: ../UpdateManager/UpdateManager.py:473
#, python-format
msgid "The package information was last updated %s day ago."
msgid_plural "The package information was last updated %s days ago."
msgstr[0] "De pakketinformatie is %s dag geleden voor het laatst bijgewerkt."
msgstr[1] ""
"De pakketinformatie is %s dagen geleden voor het laatst bijgewerkt."

#: ../UpdateManager/UpdateManager.py:477
#, python-format
msgid "The package information was last updated %s hour ago."
msgid_plural "The package information was last updated %s hours ago."
msgstr[0] "De pakketinformatie is %s uur geleden voor het laatst bijgewerkt."
msgstr[1] "De pakketinformatie is %s uur geleden voor het laatst bijgewerkt."

#: ../UpdateManager/UpdateManager.py:481
msgid "The package information was last updated less than one hour ago."
msgstr ""
"De pakketinformatie is minder dan een uur geleden voor het laatst bijgewerkt."

#: ../UpdateManager/UpdateManager.py:501 ../data/glade/UpdateManager.ui.h:23
msgid ""
"Software updates correct errors, eliminate security vulnerabilities and "
"provide new features."
msgstr ""
"Software-updates herstellen fouten, verhelpen veiligheidsproblemen en "
"leveren nieuwe mogelijkheden."

#: ../UpdateManager/UpdateManager.py:520
msgid "Welcome to Ubuntu"
msgstr "Welkom bij Ubuntu"

#: ../UpdateManager/UpdateManager.py:521
msgid ""
"These software updates have been issued since Ubuntu was released. If you "
"don't want to install them now, choose \"Update Manager\" from the "
"Administration Menu later."
msgstr ""
"Deze software-updates zijn ontwikkeld sinds Ubuntu is uitgegeven. Als u ze "
"op een later tijdstip wilt installeren, kies dan \"Updatebeheer\" uit het "
"menu Beheer."

#: ../UpdateManager/UpdateManager.py:524
msgid "Software updates are available for this computer"
msgstr "Er zijn software-updates beschikbaar voor deze computer"

#: ../UpdateManager/UpdateManager.py:525
msgid ""
"If you don't want to install them now, choose \"Update Manager\" from the "
"Administration menu later."
msgstr ""
"Als u deze updates later wilt installeren, kies dan \"Updatebeheer\" uit het "
"menu Beheer."

#: ../UpdateManager/UpdateManager.py:574
#, python-format
msgid ""
"The upgrade needs a total of %s free space on disk '%s'. Please free at "
"least an additional %s of disk space on '%s'. Empty your trash and remove "
"temporary packages of former installations using 'sudo apt-get clean'."
msgstr ""
"De update vereist %s vrije ruimte op schijf '%s'. Maak ten minste %s extra "
"ruimte vrij op '%s'. Leeg uw prullenbak, en verwijder tijdelijke "
"pakketbestanden van vorige installaties met 'sudo apt-get clean'."

#: ../UpdateManager/UpdateManager.py:631
#: ../UpdateManagerHildon/UpdateManagerHildon.py:114
msgid "Reading package information"
msgstr "Pakketinformatie lezen"

#: ../UpdateManager/UpdateManager.py:744
msgid "Could not initialize the package information"
msgstr "Kon de pakketinformatie niet initialiseren"

#: ../UpdateManager/UpdateManager.py:745
msgid ""
"An unresolvable problem occurred while initializing the package "
"information.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:\n"
msgstr ""
"Er heeft zich een onoplosbaar probleem voorgedaan bij het initialiseren van "
"de pakketinformatie.\n"
"\n"
"Rapporteer deze fout bij het pakket ‘update-manager’ en voeg de volgende "
"foutmelding toe:\n"

#: ../UpdateManager/UpdateManager.py:767
msgid ""
"An unresolvable problem occurred while calculating the upgrade.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:"
msgstr ""
"Er heeft zich een onoplosbaar probleem voorgedaan bij het berekenen van de "
"upgrade.\n"
"\n"
"Rapporteer deze fout bij het pakket ‘update-manager’ en voeg de volgende "
"foutmelding toe:"

#: ../UpdateManager/UpdateManager.py:790
msgid " (New install)"
msgstr " (Nieuwe installatie)"

#. TRANSLATORS: the b stands for Bytes
#: ../UpdateManager/UpdateManager.py:794
#, python-format
msgid "(Size: %s)"
msgstr "(Grootte: %s)"

#: ../UpdateManager/UpdateManager.py:796
#, python-format
msgid "From version %(old_version)s to %(new_version)s"
msgstr "Van versie %(old_version)s naar %(new_version)s"

#: ../UpdateManager/UpdateManager.py:800
#, python-format
msgid "Version %s"
msgstr "Versie %s"

#: ../UpdateManager/UpdateManager.py:813
msgid "Your Ubuntu release is not supported anymore"
msgstr "Uw versie van Ubuntu wordt niet meer ondersteund"

#: ../UpdateManager/UpdateManager.py:814
msgid ""
"You will not get any further security fixes or critical updates. Please "
"Upgrade to a later version of Ubuntu Linux."
msgstr ""
"U zult geen veiligheidsupdates of andere belangrijke updates meer ontvangen. "
"Upgrade uw Ubuntu Linux naar een latere versie."

#: ../UpdateManager/UpdateManager.py:821
msgid "Upgrade information"
msgstr "Upgrade-informatie"

#. print "on_button_dist_upgrade_clicked"
#: ../UpdateManager/UpdateManager.py:841 ../check-new-release-gtk:87
msgid "Downloading the release upgrade tool"
msgstr "Programma voor versie-upgrade downloaden"

#: ../UpdateManager/UpdateManager.py:848
#, python-format
msgid "<b>New Ubuntu release '%s' is available</b>"
msgstr "<b>Nieuwe Ubuntu-versie '%s' is beschikbaar</b>"

#. we assert a clean cache
#: ../UpdateManager/UpdateManager.py:887
msgid "Software index is broken"
msgstr "Software-index is beschadigd"

#: ../UpdateManager/UpdateManager.py:888
msgid ""
"It is impossible to install or remove any software. Please use the package "
"manager \"Synaptic\" or run \"sudo apt-get install -f\" in a terminal to fix "
"this issue at first."
msgstr ""
"Het is nu niet mogelijk om software te installeren of te verwijderen. "
"Gebruik het pakkettenbeheerprogramma \"Synaptic\" of gebruik \"sudo apt-get "
"install -f\" in een terminalvenster om eerst dit probleem te verhelpen."

#: ../UpdateManagerHildon/UpdateManagerHildon.py:136
msgid "Downloading Package Information"
msgstr "Pakketinformatie downloaden"

#: ../UpdateManagerHildon/UpdateManagerHildon.py:137
msgid ""
"The repositories are being checked for new, removed, or updated software "
"packages"
msgstr ""
"Er wordt gecontroleerd of er nieuwe, verwijderde of bijgewerkte "
"softwarepakketten beschikbaar zijn"

#: ../UpdateManagerHildon/UpdateManagerHildon.py:145
msgid "Downloading Package Updates"
msgstr "Pakketupdates downloaden"

#: ../UpdateManagerHildon/UpdateManagerHildon.py:146
msgid ""
"The selected package updates are being downloaded and installed on the system"
msgstr ""
"De geselecteerde pakketupdates worden gedownload en geïnstalleerd op het "
"systeem"

#: ../UpdateManagerHildon/UpdateManagerHildon.py:152
msgid "run_synaptic called with unknown action"
msgstr "run_synaptic aangeroepen met onbekende actie"

#: ../UpdateManagerText/UpdateManagerText.py:26
msgid "Cancel"
msgstr "Annuleren"

#: ../UpdateManagerText/UpdateManagerText.py:27
msgid "Install"
msgstr "Installeren"

#: ../UpdateManagerText/UpdateManagerText.py:45
msgid "Building Updates List"
msgstr "Lijst met updates genereren"

#: ../UpdateManagerText/UpdateManagerText.py:48
msgid ""
"\n"
"A normal upgrade can not be calculated, please run: \n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
"This can be caused by:\n"
" * A previous upgrade which didn't complete\n"
" * Problems with some of the installed software\n"
" * Unofficial software packages not provided by Ubuntu\n"
" * Normal changes of a pre-release version of Ubuntu"
msgstr ""
"\n"
"Er kon geen normale upgrade berekend worden, voer uit: \n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
"Dit kan veroorzaakt worden door:\n"
" * Een vorige update die niet voltooid werd\n"
" * Problemen met de geïnstalleerde software\n"
" * Onofficiële softwarepakketten die niet door Ubuntu geleverd worden\n"
" * Normale veranderingen van een vooruitgave van Ubuntu"

#: ../UpdateManagerText/UpdateManagerText.py:117
msgid "Downloading changelog"
msgstr "Wijzigingenlogboek downloaden"

#: ../UpdateManager/Core/MyCache.py:124
#, python-format
msgid "Other updates (%s)"
msgstr "Overige updates (%s)"

#: ../UpdateManager/Core/MyCache.py:246
#, python-format
msgid ""
"Changes for the versions:\n"
"%s\n"
"%s\n"
"\n"
msgstr ""
"Wijzigingen voor de versies:\n"
"%s\n"
"%s\n"
"\n"

#: ../UpdateManager/Core/MyCache.py:248
msgid "This change is not coming from a source that supports changelogs."
msgstr ""
"Deze wijziging komt niet van een bron die wijzigingenlogboeken ondersteunt."

#: ../UpdateManager/Core/MyCache.py:256
#, python-format
msgid ""
"The changelog does not contain any relevant changes.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""
"Het wijzigingenlogboek bevat geen relevante wijzigingen.\n"
"\n"
"Gebruik http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"totdat de wijzigingen beschikbaar worden, of probeer het later opnieuw."

#: ../UpdateManager/Core/MyCache.py:261
#, python-format
msgid ""
"The list of changes is not available yet.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""
"De lijst met wijzigingen is nog niet beschikbaar.\n"
"\n"
"U kunt deze wel al lezen op "
"http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"of u kunt het later nog eens proberen."

#: ../UpdateManager/Core/MyCache.py:267
msgid ""
"Failed to download the list of changes. \n"
"Please check your Internet connection."
msgstr ""
"Kon de lijst met wijzigingen niet downloaden. \n"
"Controleer uw internetverbinding."

#: ../UpdateManager/Core/UpdateList.py:50
msgid "Failed to detect distribution"
msgstr "Bepalen van de distributie is mislukt"

#: ../UpdateManager/Core/UpdateList.py:51
#, python-format
msgid "A error '%s' occurred while checking what system you are using."
msgstr ""
"Er heeft zich een fout ‘%s’ voorgedaan tijdens de controle van uw "
"systeemversie."

#: ../UpdateManager/Core/UpdateList.py:62
msgid "Important security updates"
msgstr "Belangrijke veiligheidsupdates"

#: ../UpdateManager/Core/UpdateList.py:63
msgid "Recommended updates"
msgstr "Aanbevolen updates"

#: ../UpdateManager/Core/UpdateList.py:64
msgid "Proposed updates"
msgstr "Voorgestelde updates"

#: ../UpdateManager/Core/UpdateList.py:65
msgid "Backports"
msgstr "Backports"

#: ../UpdateManager/Core/UpdateList.py:66
msgid "Distribution updates"
msgstr "Distributie-updates"

#: ../UpdateManager/Core/UpdateList.py:71
msgid "Other updates"
msgstr "Andere updates"

#: ../UpdateManager/Core/UpdateList.py:80
msgid "Previous selected"
msgstr "Eerder geselecteerd"

#: ../data/glade/UpdateManager.ui.h:1
msgid ""
"<b><big>You must check for updates manually</big></b>\n"
"\n"
"Your system does not check for updates automatically. You can configure this "
"behavior in <i>Software Sources</i> on the <i>Updates</i> tab."
msgstr ""
"<b><big>U moet zelf controleren of er updates zijn</big></b>\n"
"\n"
"Uw systeem controleert niet automatisch of er updates zijn. U kunt dit "
"instellen in <i>Softwarebronnen</i> op het tabblad <i>Updates</i>."

#: ../data/glade/UpdateManager.ui.h:4
msgid "<big><b>Not all updates can be installed</b></big>"
msgstr "<big><b>Niet alle updates kunnen worden geïnstalleerd</b></big>"

#: ../data/glade/UpdateManager.ui.h:5
msgid "<big><b>Running on battery</b></big>"
msgstr "<big><b>Werkt op accu</b></big>"

#: ../data/glade/UpdateManager.ui.h:6
msgid "<big><b>Starting Update Manager</b></big>"
msgstr "<big><b>Updatebeheer opstarten</b></big>"

#: ../data/glade/UpdateManager.ui.h:7
msgid "Changes"
msgstr "Wijzigingen"

#: ../data/glade/UpdateManager.ui.h:8
msgid "Chec_k"
msgstr "_Controleren"

#: ../data/glade/UpdateManager.ui.h:9
msgid "Co_ntinue"
msgstr "_Doorgaan"

#: ../data/glade/UpdateManager.ui.h:10
msgid "Description"
msgstr "Omschrijving"

#: ../data/glade/UpdateManager.ui.h:11
msgid "Description of update"
msgstr "Omschrijving van de update"

#: ../data/glade/UpdateManager.ui.h:13
msgid "Restart _Later"
msgstr "_Later herstarten"

#: ../data/glade/UpdateManager.ui.h:14
msgid ""
"Run a partial upgrade, to install as many updates as possible. \n"
"\n"
"This can be caused by:\n"
" * A previous upgrade which didn't complete\n"
" * Problems with some of the installed software\n"
" * Unofficial software packages not provided by Ubuntu\n"
" * Normal changes of a pre-release version of Ubuntu"
msgstr ""
"Een gedeeltelijke upgrade uitvoeren, om zoveel mogelijk updates te "
"installeren. \n"
"\n"
"Dit kan verzoorzaakt worden door:\n"
" * Een vorige upgrade die niet werd voltooid\n"
" * Problemen met de geïnstalleerde software\n"
" * Onofficiële softwarepakketten die niet ondersteund worden door Ubuntu\n"
" * Normale wijzigingen in een vooruitgave van Ubuntu"

#: ../data/glade/UpdateManager.ui.h:21
msgid "Settings..."
msgstr "Instellingen…"

#: ../data/glade/UpdateManager.ui.h:22
#: ../data/glade/UpgradePromptDialog.ui.h:5
msgid "Show progress of individual files"
msgstr "Voortgang van de afzonderlijke pakketten weergeven"

#: ../data/glade/UpdateManager.ui.h:24
msgid ""
"The computer needs to restart to finish installing updates. Please save your "
"work before continuing."
msgstr ""
"De computer moet herstart worden om het installeren van de updates te "
"voltooien. Sla u werk op alvorens u verdergaat."

#: ../data/glade/UpdateManager.ui.h:25
msgid "U_pgrade"
msgstr "U_pgraden"

#: ../data/glade/UpdateManager.ui.h:26 ../data/update-manager.desktop.in.h:3
#: ../data/update-manager-hildon.desktop.in.h:3
msgid "Update Manager"
msgstr "Updatebeheer"

#: ../data/glade/UpdateManager.ui.h:27
msgid "Your system is running on battery. Are you sure you want to continue?"
msgstr ""
"Uw systeem werkt momenteel op een accu. Weet u zeker dat u wilt doorgaan?"

#: ../data/glade/UpdateManager.ui.h:28
msgid "_Hide this information in the future"
msgstr "_Deze informatie in het vervolg niet meer tonen"

#: ../data/glade/UpdateManager.ui.h:29
msgid "_Install Updates"
msgstr "Up_dates installeren"

#: ../data/glade/UpdateManager.ui.h:30
msgid "_Partial Upgrade"
msgstr "_Gedeeltelijke upgrade"

#: ../data/glade/UpdateManager.ui.h:32
msgid "_Upgrade"
msgstr "_Upgraden"

#: ../data/glade/UpdateManager.ui.h:33
msgid "updates"
msgstr "updates"

#: ../data/glade/UpgradePromptDialog.ui.h:1
msgid ""
"<b>A new version of Ubuntu is available. Would you like to upgrade?</b>"
msgstr ""
"<b>Er is een nieuwe versie van Ubuntu beschikbaar. Wilt u upgraden?</b>"

#: ../data/glade/UpgradePromptDialog.ui.h:2
msgid "Ask Me Later"
msgstr "Later opnieuw vragen"

#: ../data/glade/UpgradePromptDialog.ui.h:3
msgid "Don't Upgrade"
msgstr "Niet upgraden"

#: ../data/glade/UpgradePromptDialog.ui.h:6
msgid "Yes, Upgrade Now"
msgstr "Ja, nu upgraden"

#: ../data/glade/UpgradePromptDialog.ui.h:7
msgid ""
"You can upgrade at a later time by opening Update Manager and click on "
"\"Upgrade\"."
msgstr ""
"U kunt op een later moment upgraden door Updatebeheer te openen en op "
"\"Upgraden\" te klikken."

#: ../data/glade/UpgradePromptDialog.ui.h:8
msgid "You have declined to upgrade to the new Ubuntu"
msgstr "U heeft besloten om niet te upgraden naar de nieuwe Ubuntu"

#: ../data/update-manager.desktop.in.h:1
#: ../data/update-manager-hildon.desktop.in.h:1
msgid "Show and install available updates"
msgstr "Beschikbare updates weergeven en installeren"

#: ../data/update-manager.desktop.in.h:2
#: ../data/update-manager-hildon.desktop.in.h:2
msgid "Software Updates"
msgstr "Software-updates"

#: ../data/update-manager.schemas.in.h:1
msgid "*deprecated* Check for new distribution releases"
msgstr "*verouderd* Controleren op nieuwe distributieversies"

#: ../data/update-manager.schemas.in.h:2
msgid "Auto close the install window after successful install"
msgstr ""
"Het installatievenster automatisch sluiten na voltooiing van de installatie."

#: ../data/update-manager.schemas.in.h:3
msgid "First run welcome message"
msgstr "Welkomstbericht bij eerste start"

#: ../data/update-manager.schemas.in.h:4
msgid ""
"If automatic checking for updates is disabled, you have to reload the "
"channel list manually. This option allows to hide the reminder shown in this "
"case."
msgstr ""
"Indien automatisch controleren voor updates uitgeschakeld is, moet u de "
"kanalenlijst zelf herladen. Met deze optie kunt u er voor zorgen dat in dat "
"soort gevallen geen herinnering gegeven wordt."

#: ../data/update-manager.schemas.in.h:5
msgid "If this key is set a first run welcome message will be presented."
msgstr ""
"Als deze optie wordt aangezet, zal er een welkomstbericht gepresenteerd "
"worden."

#: ../data/update-manager.schemas.in.h:6
msgid ""
"If this key is set the install window will be automatically closed on "
"successful installation."
msgstr ""
"Als deze optie wordt ingeschakeld, zal het installatievenster automatisch "
"gesloten worden na een succesvolle installatie."

#: ../data/update-manager.schemas.in.h:7
msgid ""
"If this key is set the main update list window will show version information "
"(from version to version)."
msgstr ""
"Bij inschakelen zal de versie-informatie weergegeven worden in de lijst met "
"updates (de wijzigingen tussen de oude versie en de nieuwe)."

#: ../data/update-manager.schemas.in.h:8
msgid "Remind to reload the channel list"
msgstr "Herinneren om de kanalenlijst te herladen"

#: ../data/update-manager.schemas.in.h:9
msgid "Show details of an update"
msgstr "Details van een update weergeven"

#: ../data/update-manager.schemas.in.h:10
msgid "Show version in update list"
msgstr "Versies weergeven in de lijst met updates."

#: ../data/update-manager.schemas.in.h:11
msgid "Stores the size of the update-manager dialog"
msgstr "Onthoudt de grootte van het dialoogvenster van updatebeheer"

#: ../data/update-manager.schemas.in.h:12
msgid ""
"Stores the state of the expander that contains the list of changes and the "
"description"
msgstr ""
"Onthoudt de toestand van het vensterdeel dat de lijst met wijzigingen en de "
"omschrijvingen bevat"

#: ../data/update-manager.schemas.in.h:13
msgid "The window size"
msgstr "De venstergrootte"

#: ../data/update-manager.schemas.in.h:14
msgid ""
"This key is deprecated in favor of the file /etc/update-manager/release-"
"upgrades Check automatically if a new version of the current distribution is "
"available and offer to upgrade (if possible)."
msgstr ""
"Deze sleutel is verouderd. Gebruik nu bij voorkeur het bestand /etc/update-"
"manager/release-upgrades.\r\n"
"Automatisch controleren of er een nieuwe versie van de huidige distributie "
"beschikbaar is en aanbieden om te upgraden (indien mogelijk)."

#: ../data/update-manager.schemas.in.h:15
msgid ""
"This will permanently hide the new release prompt from check-new-release-"
"gtk. Note that the small button in the main update-manager UI will still be "
"there."
msgstr ""
"Dit zal de melding van een nieuwe uitgave permanent verbergen voor check-new-"
"release-gtk. Merk op dat de kleine knop in de hoofd-interface van update-"
"manager nog steeds aanwezig zal zijn."

#: ../data/update-manager.schemas.in.h:16
msgid "make check-new-release-gtk ignore a given new release"
msgstr "check-release-gtk een nieuwe uitgave laten negeren"

#: ../update-manager:56 ../do-release-upgrade:33
msgid "Show version and exit"
msgstr "Softwareversie weergeven en afsluiten"

#: ../update-manager:59
msgid "Directory that contains the data files"
msgstr "Map die de gegevensbestanden bevat"

#: ../update-manager:62
msgid "Check if a new Ubuntu release is available"
msgstr "Controleren of er een nieuwe versie van Ubuntu beschikbaar is"

#: ../update-manager:65 ../do-release-upgrade:36 ../check-new-release:27
#: ../check-new-release-gtk:149
msgid "Check if upgrading to the latest devel release is possible"
msgstr ""
"Controleer of upgraden naar de nieuwste ontwikkelaarsversie mogelijk is"

#: ../update-manager:69
msgid "Upgrade using the latest proposed version of the release upgrader"
msgstr "Upgrade uitvoeren met de meest recente versie van de upgrader"

#: ../update-manager:76
msgid "Do not focus on map when starting"
msgstr "Geen focus op de map leggen tijdens opstarten"

#: ../update-manager:79
msgid "Try to run a dist-upgrade"
msgstr "Probeer om een dist-upgrade uit te voeren"

#: ../update-manager:83 ../do-release-upgrade:52
msgid "Test upgrade with a sandbox aufs overlay"
msgstr "Upgrade testen met een sandbox-aufs-overlay"

#: ../update-manager:103
msgid "Running partial upgrade"
msgstr "Gedeeltelijke upgrade uitvoeren"

#: ../do-release-upgrade:40 ../check-new-release:31
#: ../check-new-release-gtk:153
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr ""
"Probeer te upgraden naar de laatste versie door gebruik te maken van de "
"upgradesoftware van $distro-proposed"

#: ../do-release-upgrade:44
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""
"Uitvoeren in een speciale upgrademodus\n"
"Momenteel worden 'desktop' en 'server' ondersteund voor upgrades van "
"respectievelijk een desktopsysteem en een serversysteem."

#: ../do-release-upgrade:50
msgid "Run the specified frontend"
msgstr "De opgegeven 'frontend' uitvoeren"

#: ../do-release-upgrade:55
msgid ""
"Check only if a new distribution release is available and report the result "
"via the exit code"
msgstr ""
"Alleen controleren wanneer er een nieuwe distributie-uitgave beschikbaar is, "
"en het resultaat rapporteren via de afsluitstatus"

#: ../do-release-upgrade:69
msgid "Checking for a new ubuntu release"
msgstr "Controleren op een nieuwe versie van ubuntu"

#: ../do-release-upgrade:79
msgid "No new release found"
msgstr "Geen nieuwe versie gevonden"

#: ../do-release-upgrade:85 ../check-new-release:56
#, c-format
msgid "New release '%s' available."
msgstr "Nieuwe versie '%s' beschikbaar."

#: ../do-release-upgrade:86 ../check-new-release:57
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr "Voer 'do-release-upgrade' uit om naar de nieuwe versie te upgraden."

#: ../check-new-release:35
#, c-format
msgid ""
"Do not output anything, just return %s if there is a new release and %s if "
"there is none."
msgstr ""
"Geef geen uitvoer, geef %s terug als er een nieuwe versie is, en %s als die "
"er niet is."

#: ../check-new-release-gtk:67
msgid "Ubuntu %(version)s Upgrade Available"
msgstr "Upgrade naar Ubuntu %(version)s beschikbaar"

#: ../check-new-release-gtk:103
#, c-format
msgid "You have declined the upgrade to Ubuntu %s"
msgstr "U heeft besloten om niet te upgraden naar Ubuntu %s"

#: ../Janitor/computerjanitor/exc.py:30
#, python-format
msgid "Unimplemented method: %s"
msgstr "Niet-geïmplementeerde methode: %s"

#: ../Janitor/computerjanitor/file_cruft.py:46
msgid "A file on disk"
msgstr "Bestand op schijf"

#: ../Janitor/computerjanitor/package_cruft.py:45
msgid ".deb package"
msgstr ".deb-pakket"

#: ../Janitor/computerjanitor/missing_package_cruft.py:33
msgid "Install missing package."
msgstr "Ontbrekend pakket installeren"

#: ../Janitor/computerjanitor/missing_package_cruft.py:42
#, python-format
msgid "Package %s should be installed."
msgstr "Pakket %s zou geïnstalleerd moeten worden."

#. pragma: no cover
#: ../Janitor/plugins/dpkg_status_plugin.py:36
#, python-format
msgid "%i obsolete entries in the status file"
msgstr "%i verouderde bestandsvermeldingen in statusbestand"

#: ../Janitor/plugins/dpkg_status_plugin.py:39
msgid "Obsolete entries in dpkg status"
msgstr "Verouderde vermeldingen in dpkg-status"

#. pragma: no cover
#: ../Janitor/plugins/dpkg_status_plugin.py:42
msgid "Obsolete dpkg status entries"
msgstr "Verouderde vermeldingen in dpkg-status"

#: ../Janitor/plugins/kdelibs4to5_plugin.py:39
msgid ""
"When upgrading, if kdelibs4-dev is installed, kdelibs5-dev needs to be "
"installed. See bugs.launchpad.net, bug #279621 for details."
msgstr ""
"Als kdelibs4-dev geïnstalleerd is, moet kdelibs5-dev ook geïnstalleerd "
"worden om te kunnen upgraden. Zie bug #279621 op bugs.launchpad.net voor "
"details."

#: ../Janitor/plugins/langpack_manual_plugin.py:34
#, python-format
msgid "%s needs to be marked as manually installed."
msgstr "%s moet aangevinkt worden als handmatig geïnstalleerd."

#: ../Janitor/plugins/remove_lilo_plugin.py:28
msgid ""
"Remove lilo since grub is also installed.(See bug #314004 for details.)"
msgstr ""
"Lilo wordt verwijderd omdat grub ook geïnstalleerd is. (Zie bug #3314004 "
"voor meer informatie.)"
